Dockerfile

```diff
@@ -11,8 +11,10 @@ RUN apk add --update --no-cache chromium chromium-chromedriver
 RUN adduser -D auto-southwest-check-in -h /app
 USER auto-southwest-check-in
 
+ENV PATH=/app/.local/bin:$PATH
```

Comment:
I'm not sure that we need this. The `pip*` binaries are identical in the two folders and I don't think any binaries in `/app/.local/bin` are necessary for the script. Thoughts?